install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -995,6 +995,12 @@ data:
 
 {{- end }}
 
+{{- if .Values.auth.mTLS.enabled }}
+  mesh-auth-mtls-listener-port: {{ .Values.auth.mTLS.port }}
```

```suggestion
  mesh-auth-mtls-listener-port: {{ .Values.auth.mTLS.port | quote }}
```

All `ConfigMap` values must be of type string, without quoting I got:
```
Error: INSTALLATION FAILED: ConfigMap in version "v1" cannot be handled as a ConfigMap: json: cannot unmarshal number into Go struct field ConfigMap.data of type string
```